pkg/api/v1/openapi_generated.go

```diff
@@ -750,6 +750,13 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 								Format:      "",
 							},
 						},
+						"model": {
+							SchemaProps: spec.SchemaProps{
+								Description: "Interface model One of: e1000, e1000e, ne2k_pci, pcnet, rtl8139, virtio Defaults to virtio",
```

Comment:
This line might read better with a period:
`... virtio. Defaults to virtio`

Reply:
Yes but look at how other fields have the same issue. I can propose a fix for this as a follow-up if you like.

Reply:
I guess I can fix it while CI is in limbo, sec. I also posted https://github.com/kubevirt/kubevirt/pull/1188 for other cases.